### PR TITLE
[release/6.0] Disable activation host tests that fail with a 6.0 RC2 or newer SDK

### PR DESCRIPTION
**Pull request by safern**
We started seeing these fail on macos after upgrading the SDK to 6.0.100 on the release branch. Disabling as these need investigation, see: https://github.com/dotnet/runtime/pull/60256#issuecomment-958356032 for more details on the thread when we updated the SDK on main and also hit this issue.

### CODE REVIEW

File: src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.sfxproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <PlatformPackageType>TargetingPack</PlatformPackageType>
     <UseTemplatedPlatformManifest>true</UseTemplatedPlatformManifest>
-    <CreatePlatformManifest Condition="'$(PreReleaseVersionLabel)' == 'servicing'">false</CreatePlatformManifest>
+    <!-- <CreatePlatformManifest Condition="'$(PreReleaseVersionLabel)' == 'servicing'">false</CreatePlatformManifest> -->
```
Comment (Anipik) [inline src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.sfxproj:7]: this needs to be reverted i think ?

**Review by Anipik — CHANGES_REQUESTED** [review]
we need to revert the ref pack change

File: src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.sfxproj (hunk 1 above)
Comment (safern) [reply src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.sfxproj:7]: Ah you are right. I had it locally to test if this caused the test issue.

Comment (safern) [reply src/installer/pkg/sfx/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.sfxproj:7]:
```suggestion
    <CreatePlatformManifest Condition="'$(PreReleaseVersionLabel)' == 'servicing'">false</CreatePlatformManifest>
```

**Review by Anipik — APPROVED** [review]

**Comment by safern** [comment]
We will have to wait for the branch to open again for this.

**Comment by safern** [comment]
Merging since we have a small window since official builds are blocked and a PR is in queue for a fix.

**Pull request merged**